yaml/landuse.yaml

```diff
@@ -1,114 +1,482 @@
-    output: {kind: nature_reserve}
-  - filter:
-      any:
-        leisure: park
-        landuse: park
+    output: {kind: enclosure}
+  - filter: {tags->zoo: petting_zoo}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: park}
-  - filter: {leisure: garden}
-    min_zoom: GREATEST(LEAST(zoom, 16), 11)
-    output: {kind: garden}
-  - filter: {leisure: playground}
+    output: {kind: petting_zoo}
+  - filter: {tags->zoo: aviary}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: playground}
-  - filter: {leisure: golf_course}
+    output: {kind: aviary}
+  - filter: {tags->attraction: animal}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: golf_course}
-  - filter: {leisure: sports_centre}
+    output: {kind: animal}
+  - filter: {tags->attraction: water_slide}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: sports_centre}
-  - filter: {leisure: pitch}
+    output: {kind: water_slide}
+  - filter: {tags->attraction: roller_coaster}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: pitch}
-  - filter: {leisure: stadium}
+    output: {kind: roller_coaster}
+  - filter: {tags->attraction: summer_toboggan}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: stadium}
-  - filter: {leisure: common}
+    output: {kind: summer_toboggan}
+  - filter: {tags->attraction: carousel}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: common}
-  - filter: 
-      landuse: forest
-      protect_class: '6'
-    min_zoom: GREATEST(LEAST(zoom, 14), 7)
-    output: {kind: forest}
-  - filter: {landuse: forest}
-    min_zoom: GREATEST(LEAST(zoom, 14), 9)
-    output: {kind: forest}
-  - filter: {landuse: residential}
+    output: {kind: carousel}
+  - filter: {tags->attraction: amusement_ride}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: residential}
-  - filter: {landuse: retail}
+    output: {kind: amusement_ride}
+  - filter: {historic: fort}
     min_zoom: GREATEST(LEAST(zoom, 16), 9)
-    output: {kind: retail}
+    output: {kind: fort}
+
+  ############################################################
+  # TIER 1
+  ############################################################
+  - filter:
+      historic: battlefield
+      not: { operator: *us_forest_service }
+    min_zoom: *tier1_min_zoom
+    output:
+      kind: battlefield
+      tier: 1
+  - filter:
+      boundary: national_park
+      not: { operator: *us_forest_service }
+    min_zoom: *tier1_min_zoom
+    output:
+      kind: national_park
+      tier: 1
+
+  ############################################################
+  # TIER 2
+  ############################################################
+  - filter:
+      boundary: national_park
+      not: { operator: *us_parks_service }
+    min_zoom: *tier2_min_zoom
+    output:
+      kind: national_park
+      tier: 2
+  - filter:
+      any:
+        leisure: park
+        landuse: park
+      not: { operator: *us_parks_service }
```

Comment:
My filters include NOT for both `*us_parks_service` and `*us_forest_service`, but I think https://github.com/tilezen/vector-datasource/issues/987 is a better place to track that refinement.